curation_concerns-models/app/models/concerns/curation_concerns/upload_set_behavior.rb

```diff
@@ -7,7 +7,6 @@ module UploadSetBehavior
     included do
       has_many :file_sets, predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isPartOf
 
-      property :creator, predicate: ::RDF::DC.creator
```

Comment:
What will slow things down? **Writing** those ACLs or their presence?

It seems broken to me that we have all these UploadSet objects just chilling out in Fedora without any association between them and their owners. Maybe this is an orthogonal concern that should be ticketed and discussed separately with more people?